maintenance_equipment_scrap/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Maintenance Equipments Scrap',
+    'summary': 'Enhance the functionality for Scrapping Equipments',
+    'author': 'Onestein, Odoo Community Association (OCA)',
+    'website': 'http://www.onestein.eu',
+    'category': 'Human Resources',
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    'depends': [
+        'maintenance',
+        'mail',
+        'stock',
```

Comment:
I confess I was quite doubtful about using the stock settings for this, deciding for it just because there isn't a default maintenance settings functionality to use nor I thought it was the case to add it within this module.
Therefore I'm in favor of dropping the settings option for now, looking for a further module that provides a Maintenance functionality.